azure/Guidelines.md

```diff
@@ -757,68 +761,92 @@ implemented as a _long-running operation (LRO)_. This allows clients to continue
 operation is being processed. The client obtains the outcome of the operation at some later time
 through another API call.
 See the [Long Running Operations section](./ConsiderationsForServiceDesign.md#long-running-operations) in
-Considerations for Service Design for an introduction to the design of long running operations.
+Considerations for Service Design for an introduction to the design of long-running operations.
 
 :white_check_mark: **DO** implement an operation as an LRO if the 99th percentile response time is greater than 1s.
 
+:no_entry: **DO NOT** implement PATCH as an LRO.  If LRO update is required it should be implemented with POST.
```

Comment:
```suggestion
:no_entry: **DO NOT** implement PATCH as an LRO.  If LRO update is required it must be implemented with POST.
```
Let's have the language match--DO NOT = MUST (as compared to the weaker 'should')